### DISCUSSION TITLE
Filter out completed kata?

### DISCUSSION THREAD

**Discussion by KevinRuggles** [codewars/codewars.com#2361] [Q&A] [answered]
If there is a way to filter kata you have done, I would like to know that method.
If there is not, consider this a suggestion for additional feature to do so.
thank you.

**Comment by kazk** [comment, marked as answer]
You can filter by progress:

![image](https://user-images.githubusercontent.com/639336/114948964-eab94400-9e04-11eb-9018-d24a7ede087d.png)

https://www.codewars.com/kata/search/my-languages?q=&&xids=completed

We don't support filtering completion by language at the moment, and it's been requested.
